### PR TITLE
[INLONG-1734][feature][audit] audit-sdk module

### LINKED ISSUE #1734: [Feature]  [InLong  audit] AuditSdk
### Description

 This is a sub issue for releasing InLong audit.
image

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by doleyzi**
add audit-sdk module
Fixes #1734

### CODE REVIEW

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderChannel.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/**
```
Comment (gosonzhang) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderChannel.java]: The implementation of this part is relatively small, can you write one yourself?

**Review by gosonzhang — CHANGES_REQUESTED** [review]

**Review by gosonzhang — APPROVED** [review]
+1

File: inlong-audit/audit-sdk/pom.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.4</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>audit-common</artifactId>
+            <version>0.12.0-incubating-SNAPSHOT</version>
```
Comment (baomingyu) [inline inlong-audit/audit-sdk/pom.xml:65]: <version>0.12.0-incubating-SNAPSHOT</version>  this version use <version>${project.version}</version> please

File: inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/AuditImp.java [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+    /**
+     * init
+     */
+    private void init() {
+        if (inited) {
+            return;
+        }
+        config.init();
+        timer.schedule(timerTask, 1000 * 60, 1000 * 60);
```
Comment (baomingyu) [inline inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/AuditImp.java:80]: logic number '1000 * 60, 1000 * 60'

**Pull request closed without merging**